src/functionalTest/java/uk/gov/hmcts/dts/fact/ServiceAreaEndpointTest.java

```diff
@@ -16,17 +15,7 @@
 
 @ExtendWith({SpringExtension.class})
 @SpringBootTest(classes = {OAuthClient.class})
```

Comment:
Just a quicky one this one. The extended class also has the same annotation, is it needed on both? And what affect does this have?

Reply:
The extended class hasn't got the same annotation for this one. However the OAuthClient.class is redundant for this one as this endpoint does not need to be authenticated. I will remove that